src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs

```diff
@@ -38,7 +38,7 @@
 (defn- ensure-sequence [input]
   ;;input may or maynot be a sequence, make it a sequence
   (cond (sequential? input) input
-        (nil? input) []
+        (or (= input "") (nil? input)) []
```

Comment:
Is `[""]` a possible value for `input`?

Didn't happen in the current scenario, but we wouldn't want to have to revisit this code.